airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```
@@ -217,10 +217,31 @@ def _make_handled_request(self) -> Any:
+                response_json = response.json()
+            except Exception as e:
+                # if the json could not be parsed, save the exception to raise it later
+                json_exception = e
+
+            try:
+                if response_json:
+                    # extract the access token and add to secrets to avoid logging the raw value
+                    access_key = self._extract_access_token(response_json)
+                    if access_key:
+                        add_to_secrets(access_key)
+            except ResponseKeysMaxRecurtionReached as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)